config/locales/server.en.yml

```
@@ -7,6 +7,7 @@ en:
     oauth2_client_id: 'Client ID for custom OAuth2'
     oauth2_client_secret: 'Client Secret for custom OAuth2'
     oauth2_authorize_url: 'Authorization URL for OAuth2'
+    oauth2_authorize_signup_url: '(optional) Alternate authorization URL used when the "Sign Up" button is used'
```

"Alternate" is a verb, right?

```suggestion
    oauth2_authorize_signup_url: '(optional) Alternative authorization URL used when the "Sign Up" button is used'
```

---

`alternate` can be a noun, verb or an adjective, and it is pronounced differently depending on the context. Yay for the English language 😂 

https://www.vocabulary.com/articles/chooseyourwords/alternate-alternative/

But I agree 'Alternative' is more clear, let's change to that.